assets/data-port/import/data/normalizer.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Normalize uploads state.
+ *
+ * @param {Object} files Files raw data.
+ *
+ * @return {Object} Normalized levels data.
+ */
+export const normalizeUploadsState = ( files ) => {
+	const levels = {};
+
+	Object.entries( files ).forEach( ( pair ) => {
```

Comment:
Looks a bit nicer with destructuring `( pair) => ` to something like ` ( [ level, file ] ) => `.

Reply:
Good shout 6f4c8b1c102a8d02ea191177bd28ce5e269b61bf